app/templates/components/component-tree-item.hbs

```diff
@@ -21,10 +21,26 @@
   {{/if}}
 
   <code
-    class="component-tree-item__tag {{if @item.isComponent "component-tree-item__bracket"}} {{if (not @item.hasChildren) "component-tree-item__bracket--self-closing"}}"
+    class="component-tree-item__tag 
+    {{if (and (not @item.args.positional) @item.isComponent) "component-tree-item__bracket"}} 
+    {{if (and (not @item.args.positional) (not @item.hasChildren)) "component-tree-item__bracket--self-closing"}}
+    {{if (and @item.args.positional @item.isComponent) "component-tree-item-classic__bracket"}} 
+    {{if (and @item.args.positional (not @item.hasChildren)) "component-tree-item-classic__bracket--self-closing"}}"
   >
     {{#if @item.isComponent}}
-      {{classify @item.name}}
+      {{#if @item.args.positional}}
+        {{dasherize @item.name}}
+      {{else}}
+        {{classify @item.name}}
+      {{/if}}
+
+      {{#each-in @item.args.positional as |name value|}}
```

Comment:
`#each`

Reply:
Why `#each`? `#each-in` is working here, I assumed because it is an object and not an array? Perhaps I am wrong though.

Reply:
It is an array. You _can_ use `{{each-in}}` on an array, for the same reason you _can_ use `for(... in ...)` on an array, which is that the array index are "properties" on the array "object". But it's definitely strange and suboptimal, so should be avoided.